Marlin/Marlin_main.cpp

```diff
@@ -6195,11 +6251,16 @@ inline void gcode_M109() {
 
     target_extruder = active_extruder; // for print_heaterstates
 
+    #if ENABLED(PRINTER_EVENT_LEDS)
+      const float start_temp = thermalManager.degTargetBed();
+      uint8_t old_red = 255;
```

Comment:
Ah, right… `thermalManager.degBed` is better!

Since `old_red` is altered below, it can't be `const`. And it should be 255 to start, because this ensures the color will be set on the first iteration.